docs/articles/documentation/reference/api/runner/clientscripts.md

```diff
@@ -13,29 +13,97 @@ async clientScripts( script[, script2[, ...[, scriptN]]] ) → this
 
 Parameter | Type                | Description
 --------- | ------------------- | ------------
-`script`, `script2`, `scriptN`  | String &#124; Object &#124; Array | Scripts to inject into the tested pages. See [Provide Scripts to Inject](../../../guides/advanced-guides/inject-client-scripts.md#provide-scripts-to-inject) to learn how to specify them.
+`script`, `script2`, `scriptN`  | String &#124; Object &#124; Array | Scripts to inject into the tested pages.
```

Comment:
To editors: no text was added or modified in this file except for the headers. Other changes are technical.